### ISSUE TITLE
While navigating with tab key, Keyboard focus is going on the non-interactive text present in the 'Azure database for mySQL' dialog:A11y_MySQLextensionforAzureDataStudio_New deployment_Usablity

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio-mysql#255]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHub Tags:**
#A11yUsable; #A11yTCS;  #DesktopApp; #BM_MySQLextensionforAzureDataStudio_Win32_Mar25; #MySQL extension for Azure Data Studio; #Win32; #Benchmark; #Win11;

**Environment and OS details:**
Azure Data Studio Version: 1.51.1 (user setup)
Windows 11 Enterprise Version 24H2
OS build 26100.3323

**Repro Steps:**
1.Launch Azure Data Studio.
2.Tab to the Connections icon (if not already selected) from left bar and press enter.
3.Tab to more button and enter on new deployment 
4.In the Deployment options, select "Azure Database For MySQL" card and press enter
5.Check using tab key focus going to noninteractive control or not.

**Actual Result:**
While navigating with tab key, Keyboard focus is going on the non-interactive text present in the 'Azure database for mySQL' dialog

**Expected Result:**
While navigating with tab key, Keyboard focus should be going on the interactive controls only.

**Attachment:**

https://github.com/user-attachments/assets/c4018d8c-7082-436d-81cd-2c212d76f2e8
